### PR TITLE
Reviews section

### PR DESCRIPTION
**Pull request by DayOut**

### CODE REVIEW

File: src/css/sections/reviews.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
+.review-cards-list {
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+}
+
+@media only screen and (min-width: 767.99px) {
```
Comment (VOSolyanik) [inline src/css/sections/reviews.css]: (min-width: 768px)

File: src/css/sections/reviews.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
+.review-cards-list {
```
Comment (VOSolyanik) [inline src/css/sections/reviews.css:1]: in general, we are using 2 spaces indents. Please, format the file

File: src/css/sections/reviews.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    }
+}
+
+.review-card {
+    background-color: var(--gray-color);
+    border-radius: 16px;
+    padding: 20px;
+}
+
+@media only screen and (min-width: 767.99px) {
```
Comment (VOSolyanik) [inline src/css/sections/reviews.css]: (min-width: 768px)

File: src/css/sections/reviews.css [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    .review-card {
+        max-width: 320px;
+    }
+}
+
+.review-card:not(:first-child) {
+    display: none;
+}
+
+@media only screen and (min-width: 767.99px) {
```
Comment (VOSolyanik) [inline src/css/sections/reviews.css]: (min-width: 768px)

File: src/partials/reviews.html [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    <ul class="review-cards-list">
+      <li class="review-card">
+        <div class="review-header">
+          <picture>
+            <source
+              srcset="
+                images/reviews/anna-des.png    1x,
+                images/reviews/anna-des@2x.png 2x
+              "
+              media="(min-width: 1280px)"
```
Comment (VOSolyanik) [inline src/partials/reviews.html]: we are using 1152 in CSS, maybe we should use that value here too?

File: src/css/sections/reviews.css [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+.review-header {
+    margin-bottom: 20px;
+    padding: 14px 14px;
+    border-radius: 8px;
+    background-color: var(--green-color);
+    display: flex;
+}
+
+.review-photo {
+    max-width: 50px;
```
Comment (VOSolyanik) [inline src/css/sections/reviews.css]: let's use the jpg versions of images, without rounding, and add rounding with border-radius and overflow.
JPG will take less size than png, so app will load faster.

File: src/css/sections/reviews.css [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+    font-family: Manrope;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 16px;
+    text-align: left;
+}
+
+.review-text {
+    color: rgba(18, 20, 23, 0.5);
+    font-family: Manrope;
```
Comment (VOSolyanik) [inline src/css/sections/reviews.css]: not needed, set globally

File: src/css/sections/reviews.css [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+    font-size: 20px;
+    font-weight: 700;
+    line-height: 24px;
+    letter-spacing: -0.02em;
+    text-align: left;
+}
+
+.review-position {
+    color: #FFFFFF80;
+    font-family: Manrope;
```
Comment (VOSolyanik) [inline src/css/sections/reviews.css]: not needed

**Review by VOSolyanik — CHANGES_REQUESTED** [review]

File: src/css/sections/reviews.css (hunk 6 above)
Comment (DayOut) [reply src/css/sections/reviews.css]: How did you download image without rounding?

File: src/css/sections/reviews.css (hunk 1 above)
Comment (DayOut) [reply src/css/sections/reviews.css]: Fixed

File: src/css/sections/reviews.css (hunk 2 above)
Comment (DayOut) [reply src/css/sections/reviews.css:1]: Missed default formatter 🤦  Fixed

File: src/css/sections/reviews.css (hunk 3 above)
Comment (DayOut) [reply src/css/sections/reviews.css]: Fixed

File: src/css/sections/reviews.css (hunk 4 above)
Comment (DayOut) [reply src/css/sections/reviews.css]: Fixed

File: src/partials/reviews.html (hunk 5 above)
Comment (DayOut) [reply src/partials/reviews.html]: Yep, i found that we using 1152 later... Fixed

File: src/css/sections/reviews.css (hunk 7 above)
Comment (DayOut) [reply src/css/sections/reviews.css]: Fixed

File: src/css/sections/reviews.css (hunk 8 above)
Comment (DayOut) [reply src/css/sections/reviews.css]: Fixed

File: src/css/sections/reviews.css (hunk 7 above)
Comment (VOSolyanik) [reply src/css/sections/reviews.css]: Not fixed

**Review by VOSolyanik — APPROVED** [review]

**Pull request merged**